sqlwrapper/ghostferry_db.go

```diff
@@ -86,3 +88,7 @@ func (tx Tx) QueryRowContext(ctx context.Context, query string, args ...interfac
 func (tx Tx) QueryRow(query string, args ...interface{}) *sqlorig.Row {
 	return tx.Tx.QueryRow(query, args...)
 }
+
+func Comment(query, comment string) string {
```

Comment:
I think `annotate` is a better name here. It doesn't need to be exported so it can be lowercased.